controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -125,7 +126,7 @@ func (r *StorageConsumerReconciler) initReconciler(request reconcile.Request) {
 	r.cephBlockPool.Namespace = r.namespace
 
 	r.cephFilesystemSubVolumeGroup = &rookCephv1.CephFilesystemSubVolumeGroup{}
-	r.cephFilesystemSubVolumeGroup.Name = fmt.Sprintf("%s-%s", "cephFilesystemSubVolumeGroup", r.storageConsumer.Name)
+	r.cephFilesystemSubVolumeGroup.Name = fmt.Sprintf("%s-%s", "cephfilesystemsubvolumegroup", r.storageConsumer.Name)
```

Comment:
you also need to set the filesystem name or else it will fail. @leseb am I correct?

Reply:
Correct, `FilesystemName` is a mandatory field.

Reply:
This has been taken care in the `reconcileCephFilesystemSubVolumeGroup` method